BioArchLinux/r-igraph/PKGBUILD

```diff
@@ -27,9 +27,6 @@ optdepends=(
   r-tcltk
   r-testthat
   r-withr
-  gmp
-  glpk
-  libxml2
 )
```

Comment:
@starsareintherose  please add description to these optional depends

Reply:
this is generated by python script, commit on pkgbuild-generator

Reply:
> this is generated by python script, commit on pkgbuild-generator

nope, they are introduced by you, see https://github.com/BioArchLinux/Packages/commit/082fb4cb090ec3ee2865d5ec2ff9b16e0b8c183f

Reply:
Oh I see, they are in the top of the PKGBUILD and marked as optional